loader/loader.go

```diff
@@ -620,10 +620,6 @@ func (l *Loader) Process(ctx context.Context, pr chan pb.ProcessResult) {
 	default:
 	}
 
-	if len(errs) != 0 {
-		// pause because of error occurred
-		l.Pause()
-	}
 	pr <- pb.ProcessResult{
```

Comment:
just a question, in old logic loader and syncer would pause when meet error but relay unit not, can you tell me why the relay is different from the other unit?